stable/neo4j/templates/tests/test-neo4j-cluster.yaml

```diff
@@ -9,13 +9,16 @@ metadata:
     app: {{ template "neo4j.name" . }}
   annotations:
     "helm.sh/hook": test-success
+    "helm.sh/hook-delete-policy": "before-hook-creation,hook-succeeded"
```

Comment:
Yes I am aware of the bug, but leaving this here in hopes that it will be fixed at some point. Once that is done, this is the right delete policy IMHO.